Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs

```diff
@@ -245,8 +237,9 @@ protected virtual void UnregisterManagers()
         /// </summary>
         /// <param name="previousObject">Object that was previously being focused.</param>
         /// <param name="newObject">New object being focused.</param>
-        protected virtual void OnFocusedObjectChanged(GameObject previousObject, GameObject newObject)
+        public virtual void OnFocusChanged(FocusEventData eventData)
```

Comment:
It changed from an event handler to a method called by the InputManager but I think you're right, it should stay protected.